### ISSUE TITLE
simplify hqx?

### ISSUE THREAD

**Issue by zvezdochiot** [bbbbbr/gimp-plugin-pixel-art-scalers#12]
See:
* https://github.com/ImageProcessing-ElectronicPublications/photoquick-plugins/tree/master/src/transform/pixart-scaler
* https://github.com/ImageProcessing-ElectronicPublications/photoquick-examples/tree/main/plugins/transform/upscale

**Comment by bbbbbr** [comment]
Can you summarize the suggestion? Is it to use the single source file hqx?

**Comment by zvezdochiot** [comment]
@bbbbbr say:
>Is it to use the single source file hqx?

* https://github.com/ImageProcessing-ElectronicPublications/photoquick-plugins/blob/master/src/transform/pixart-scaler/scaler_hqx.c

Header for all scaler:

* https://github.com/ImageProcessing-ElectronicPublications/photoquick-plugins/blob/master/src/transform/pixart-scaler/scaler.h

---

PS:
https://github.com/bbbbbr/gimp-plugin-pixel-art-scalers/blob/2c79cf2272c4c1e05e0edf20bee9098812af0d34/src/hqx_init.c#L22-L38

⚠️ Init is completely unnecessary. It only slows down and eats away memory. 

ℹ️ See also:
* #10 
* https://github.com/bbbbbr/gimp-plugin-pixel-art-scalers/pull/1/commits/8bfff31328c1e3a42c1cee8af9e6c4ff46f4a7d5

**Comment by zvezdochiot** [comment]
Overdue.
